bot/helper/mirror_utils/download_utils/direct_link_generator.py

```diff
@@ -54,12 +54,14 @@ def direct_link_generator(link: str):
         return anon(link)
     elif 'letsupload.io' in link:
         return letsupload(link)
-     elif 'fembed.net' in link:
+    elif 'fembed.net' in link:
         return fembed(link)
     elif 'fembed.com' in link:
         return fembed(link)
     elif 'femax20.com' in link:
         return fembed(link)
+    elif 'fcdn.stream' in link:
```

Comment:
this is different ma not work with fembed

Reply:
Works
- [ ] 
![Screenshot_20210704-194243](https://user-images.githubusercontent.com/71178188/124385485-13fcb880-dd00-11eb-85e4-4a7069c6df06.png)

Reply:
> Works
> ![Screenshot_20210704-194243](https://user-images.githubusercontent.com/71178188/124385485-13fcb880-dd00-11eb-85e4-4a7069c6df06.png)

Ahh nice i didn't tested!

Reply:
try this mixdrop `https://mixdrop.co/f/zprkx17nsgmme4m`

Reply:
Well, lk21 doesn't support mixdrop, so ig it will not work

Reply:
> Well, lk21 doesn't support mixdrop, so ig it will not work

leave... will see alt way for it.